src/models/user.d.ts

```diff
@@ -105,6 +112,7 @@ declare namespace rest_api {
         tournaments: Array<{
             id: number;
             name: string;
+            icon: string; // URL
```

Comment:
This is good, I also added `icon` to `ladders` up above if you want to add that in. 

Down below we still have `groups: any[]` - this can be array of `                {
                    "id": number;
                    "name": string;
                    "icon": string;
                }
`
too

and while we're at it `titles` is `
                {
                    "icon": string;
                    "title": string;
                }
`

Reply:
Added.